ios/Exponent/ExpoKit/ExpoKit.m

```diff
@@ -112,9 +112,10 @@ - (void)application:(UIApplication *)application didFinishLaunchingWithOptions:(
       }
     }
   }
-
+  
   // This is safe to call; if the app doesn't have permission to display user-facing notifications
   // then registering for a push token is a no-op
+  [[EXUserNotificationManager sharedInstance] authorizeAndInit:launchOptions];
```

Comment:
Is this safe to call? It looks like it will ask the user for permission to show notifications. It also needs to run on the notifications queue for iOS 10 & 11.